content/en/docs/tutorials/kubernetes-basics/_index.md

```diff
@@ -17,95 +17,95 @@
 
 <div class="layout" id="top">
 
-  <main class="content">
+ <main class="content">
 
-    <div class="row">
-      <div class="col-md-9">
-        <h2>Kubernetes Basics</h2>
-        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
-        <p>Using the interactive tutorials, you can learn to:</p>
-        <ul>
-        <li>Deploy a containerized application on a cluster.</li>
-        <li>Scale the deployment.</li>
-        <li>Update the containerized application with a new software version.</li>
-        <li>Debug the containerized application.</li>
-        </ul>
-        <p>The tutorials use Katacoda to run a virtual terminal in your web browser that runs Minikube, a small-scale local deployment of Kubernetes that can run anywhere. There's no need to install any software or configure anything; each interactive tutorial runs directly out of your web browser itself.</p>
+  <div class="row">
+     <div class="col-md-9">
+       <h2>Kubernetes Basics</h2>
+       <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
+       <p>Using the interactive tutorials, you can learn to:</p>
+       <ul>
+       <li>Deploy a containerized application on a cluster.</li>
+       <li>Scale the deployment.</li>
+       <li>Update the containerized application with a new software version.</li>
+       <li>Debug the containerized application.</li>
+       </ul>
+       <p>The tutorials use Katacoda to run a virtual terminal in your web browser that runs Minikube, a small-scale local deployment of Kubernetes that can run anywhere. There's no need to install any software or configure anything; each interactive tutorial runs directly out of your web browser itself.</p>
       </div>
-    </div>
+     </div>
 
-    <br>
```

Comment:
In the preview, https://deploy-preview-38834--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/, everything from here to the end of the page is rendered in a code block, not has HTML. I assume there is some weird Markdown rendering happening where it things this is a code block because of spacing, but i'm not sure of the exact cause

Reply:
Alright. Let me check.
We always had such problems when nested HTML has 4 spaces indentation.

Reply:
fixed